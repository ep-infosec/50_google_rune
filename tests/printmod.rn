//  Copyright 2021 Google LLC.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

println 6 + 6 mod 7u128
println -1 mod 123
println 5*5 mod 13u128
