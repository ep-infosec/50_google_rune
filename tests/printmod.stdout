5
122
12
